lib/experiments/index.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 var symbol = require('../utilities/symbol');
+var version = require('../../package').version;
+var isCanaryVersion = version.match(/-alpha/);
```

Comment:
I'm tempted to label it `2.12.0-canary` since we don't actually publish individual releases. I've not though through what that would mean if we wanted to actually do a release off of that branch. It's too late at night for that.